mapping.md

```diff
@@ -0,0 +1,228 @@
+# Data model mapping between Neo and NIX
+
+## neo.Block
+
+Maps directly to `nix.Block`.
+  - Attributes
+
+    | Neo                           | NIX                                  |
+    |-------------------------------|--------------------------------------|
+    | Block.name(string)            | Block.name(string)                   |
+    | Block.description(string)     | Block.definition(string)             |
+    | Block.rec\_datetime(datetime)  | Block.created\_at(int)                |
+    | Block.file\_datetime(datetime) | Block.metadata(**Section**) [[1]](#notes) |
+    | Block.file\_origin(string)     | Block.metadata(**Section**) [[1]](#notes) |
```

Comment:
The `file_origin` must only be stored in the corresponding metadata section, if `file_origin` is meant to be the original file and not current file. If the latter is the case it is not necessary to store it at all.